src/routes/expenses.routes.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const express = require('express');
+const cors = require('cors');
+
+const expensesController = require('../controllers/expenses.controller');
+
+const expensesRouter = express.Router();
+
+expensesRouter.use(cors());
```

Comment:
This is not needed in this task, has no impact on the workflow, so I would delete that.
Also you do it just once in the createServer and that's enough